apps/schematic/api/schematic_api/controllers/versions_controller_impl.py

```diff
@@ -0,0 +1,15 @@
+"Version endpoint controllers"
+
+import importlib.metadata
+
+from schematic_api.controllers.utils import handle_exceptions
+
+
+@handle_exceptions
+def get_schematic_version() -> str:
+    """Gets the current schematic version
+
+    Returns:
+        str: The current schematic version
+    """
+    return importlib.metadata.version("schematicpy")
```

Comment:
Does this give an accurate result? I ask because checking the version of a locally installed schematic is kind of funky at the moment. I don't think the version spec in the `.toml` is consistently updated and schematic doesn't have functionality to check its own version. I recognize that this is not ideal; there is an [issue open to remedy](https://sagebionetworks.jira.com/browse/FDS-831). 

For the api purposes right now it's specified thorough an environment variable in the docker image. If using that doesn't work and it must be checked this way then @AmyHeiser we should prioritize the above linked issue.

Reply:
Here's the [documentation](https://docs.python.org/3/library/importlib.metadata.html.)

It does, it returns `23.7.1` which is the version of schematic it's currently using

Reply:
@GiaJordan I'm not sure if this would work for getting the version of "itself" that seems complicated.

Reply:
Can you send the link again please? I'm getting a `404 Error`

Reply:
> It does, it returns `23.7.1` which is the version of schematic it's currently using

cool, thanks for confirming!

Reply:
Here's the link:

Reply:
Thanks, that's what I was looking at too. I think the difference is you've installed schematic with `pip` and I have the developers install.